src/SwedbankPay.Sdk/PaymentInstruments/Invoice/InvoicePayment.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace SwedbankPay.Sdk.PaymentInstruments.Invoice
+{
+    public class InvoicePayment : IInvoicePayment
```

Comment:
If this class is only used for requests, I think `InvoicePaymentRequestData` is appropriate.

```suggestion
    public class InvoicePaymentRequestData : IInvoicePayment
```